MetaMask/metamask-extension
# Fix for contacts tab alignment bugs

## Explanation

Fixed alignment of title in Contacts bug, removed spaces in public addresses, fixed positioning of labels and scroll element and added logic for Save button to be disabled when nothing is changed.
* Fixes #15887

## Screenshots/Screencaps

### Before

![contactsTabBug](https://user-images.githubusercontent.com/120601632/212307612-a3b1beb3-f2f8-4bfc-8dc9-9954a057b520.gif)

### After

![contactsTabBugFix](https://user-images.githubusercontent.com/120601632/212307267-d57fc9cf-d69e-456d-a75f-76535454366b.gif)

## Manual Testing Steps
First scenario:

1. Open MM
2. Go to Settings -> Contacts
3. Add one contact to address book (add username and public address)
4. Open added contact
5. Check ethereum public address
6. Check alignment of address-book header name
7. Click Edit
8. Labels (Username, Ethereum public address, Memo) are now positioned well
9. Scroll element in Ethereum public address input field is now positioned well
10. Public address is now inside input field when I minimize window size

Second scenario:

1. Open MM
2. Go to Settings -> Contacts
3. Save button is now disabled when nothing is changed
4. Add one contact to address book with long username
5. Open added contact and minimize window, header is now aligned and text is not overflowing
6. On extension popup we now show address/username in header
